pkg/pgmodel/ingestor/dispatcher.go

```diff
@@ -167,6 +170,9 @@ func (p *pgxDispatcher) getServerEpoch() (model.SeriesEpoch, error) {
 }
 
 func (p *pgxDispatcher) CompleteMetricCreation(ctx context.Context) error {
+	if p.closed.Load() {
+		return fmt.Errorf("dispatcher is closed")
```

Comment:
This error is repeated. I think we should declare as follows. This allows the caller to use the error in some future case.
```golang
var ErrDispatcherClosed = fmt.Errorf("dispatcher is closed")
```